app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -217,4 +217,39 @@ class SmokeTest {
             }
         }
     }
+
+    @Test
+    fun verifyETPToolbarShieldIconIsNotDisplayedIfETPIsOFFGloballyTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openEnhancedTrackingProtectionSubMenu {
+            clickEnhancedTrackingProtectionDefaults()
+            verifyEnhancedTrackingProtectionOptionsGrayedOut()
+        }.goBackToHomeScreen {
+            navigationToolbar {
+            }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+                verifyEnhancedTrackingProtectionPanelNotVisible()
+            }.openThreeDotMenu {
+            }.clickAddOnsReportSiteIssue {
+                verifyUrl("webcompat.com/issues/new")

```

Comment:
I think we are using this in other tests too..could we add a variable somewhere so that in case it change it would be easier to change only in one place?

Reply:
yeah this was supposed to be in AddOns robot that Kadeem has been working on.